tests/oisp-tests.js

```diff
@@ -41,22 +41,42 @@ var actuatorType = "powerswitch.v1.0"
 var switchOnCmdName = "switch-on"
 var switchOffCmdName = "switch-off"
 
+var recipientEmail = "jamal.el.youssefi@intel.com"

```

Comment:
@jelyoussefi please use an anonymous email (even though it will not be sent, it should be e.g. test.receiver@streammyiot.com.